src/common/OperationsBase.ts

```diff
@@ -1,20 +1,26 @@
 /*---------------------------------------------------------------------------------------------
```

Comment:
Why are there so many changes in this file that are not related to the thing you are adding? Revert all the unnecessary changes.

Reply:
Revert all the unnecessary in all files. It's really hard to see what you actually changed.